src/store/reducers/appReducer.js

```diff
@@ -3,39 +3,39 @@
 import 'rxjs';
 import { createAction, createReducerScenario, createEpicScenario } from 'store/utilities';
 
-// ========================= Reducers - START
-const testReducer = (state, action) => ({ ...state, testData: action.payload });
-
-export const redux = createReducerScenario({
-  updateDeviceModels: { type: 'TEST_DATA_UPDATE', reducer: testReducer }
-});
-
-export const reducer = { app: redux.getReducer() };
-// ========================= Reducers - END
-
-// ========================= Selectors - START
-// ========================= Selectors - END
-
 // ========================= Epics - START
 export const epics = createEpicScenario({
   /** Initializes the redux state */
   initializeApp: {
     type: 'APP_INITIALIZE',
     epic: () => [
-      redux.actions.updateDeviceModels(),
+      // redux.actions.updateDeviceModels(),
     ]
   },
 
   /** Listen to route events and emit a route change event when the url changes */
   detectRouteChange: {
     type: 'APP_ROUTE_EVENT',
     rawEpic: (action$, store, actionType) =>
-      action$.ofType(actionType)
-        .map(({ payload }) => payload) // Pathname
+      action$
+        .ofType(actionType)
+        .map(({ payload }) => payload) // payload === pathname
         .distinctUntilChanged()
-        .do(pathname => console.log('appReducer', pathname))
         .map(createAction('EPIC_APP_ROUTE_CHANGE'))
   }
 
 });
 // ========================= Epics - END
+
+// ========================= Reducers - START
+const testReducer = (state, action) => ({ ...state, testData: action.payload });
+
+export const redux = createReducerScenario({
```

Comment:
Are these "test" things needed now that we're adding real ones?

Reply:
Nope but this will be removed in the next PR